src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,112 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private V[] arrayValue = (V[]) new Object[0];
+    private K[] arrayKey = (K[]) new Object[0];
+    private V nullValue = null;
```

Comment:
<img width="1133" alt="image" src="https://user-images.githubusercontent.com/81781479/206193145-1033a8ea-3cb0-4211-ab74-7a24808b7a92.png">

```suggestion
    private static final int MAX_SIZE = 10;
    private V[] valuesArray;
    private K[] keysArray;
    private int size;
    
    pubic StorageImpl() {
        keysArray = (K[]) new Object[MAX_SIZE];
        valuesArray = (V[]) new Object[MAX_SIZE];
    }
```
давай спростимо трохи собі роботу)) 
В завданні гарантовано що кількість елементів буде максимум 10 - можна одразу створити масиви на 10 елементів (колекцію яка динамічно буде збільшуватись будеш імплементувати в завданні array-list, linked-list, hash-map)

Поле `size` буде завжди мати актуальний розмір стореджа
Також `size` буде завжди наступним вільним місцем для наступних елементів. Тобто якщо size = 3, то останній елемент зберігається під індексом 2 (бо в таблицях індексація йде від 0) а наступний вільний слот буде під індексом 3 тобто size